api/snippets-menu/README.md

```diff
@@ -0,0 +1,52 @@
+# Snippets Menu
+
+AoU Clusters enable the [Snippets Menu extension](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/snippets_menu/readme.html)
+with custom AoU-specific code snippets. The snippets live in
+https://github.com/all-of-us/workbench-snippets. AoU configures Leo clusters as
+follows to enable this:
+
+1. Enable the snippets_menu/main extension in ./initialize_notebook_cluster.sh
+1. Deploy a Jupyter UI extension to configure the menu with AoU-specific snippets
+
+## Updating Snippets
+
+As code snippets are updated in the source repository, we will want to
+periocially update the menus in the Workbench. For now, this process is fairly
+manual and can be improved going forwards (see RW-2665):
+
+Prerequisite: Must have `jq` installed (for pretty printing).
+
+1. In a separate directory, clone the snippets repo and build the menu config:
+    https://github.com/all-of-us/workbench-snippets/blob/master/CONTRIBUTING.md#auto-generation-of-jupyter-snippets-menu-configuration
+1. Run the following from the `workbench/api/cluster-resources` directory to pull in the updated JSON
+
+    ```
+    ./import_json_from_snippets_repo.sh <path to workbench-snippets repo>
```

Comment:
This script needs to be updated to copy to the correct destination.  output: 
```
Copied pretty printed ../../../aou-snippets/build/r_dataset_snippets_menu_config.json into /Users/thibault/src/aou/api/cluster-resources/r-dataset-snippets-menu.json
```

Reply:
Oops, thanks. I've now run this myself to validate.